<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="setTemperature_pushButton">
       <property name="text">
        <string>set</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IP Address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Quick Connect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="quickConnect_comboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Montana 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Montana 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QPushButton" name="disconnect_pushButton">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ipaddress_lineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>136.167.55.165</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QPushButton" name="connect_pushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QTextEdit" name="logStatus_textEdit"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Temperature Status</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Current Temperature</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="currentTemperature_label">
       <property name="text">
        <string>300.0 K</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="temperatureStatus_label">
       <property name="text">
        <string>Stable</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="bufferTime_spinBox">
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Post Stablizing Buffer (s)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Temperature Target (K)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="targetTemp_doubleSpinBox">
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string> K</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>350.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Current Target</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="getTemperature_pushButton">
       <property name="text">
        <string>get</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="targetTemperature_label">
       <property name="text">
        <string>300.0 K</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Max Stablizing Wait Time</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="maxStablizingWait_spinBox">
       <property name="suffix">
        <string> min</string>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QPushButton" name="stopwaiting_pushButton">
       <property name="text">
        <string>stop waiting stable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
